src/icons/icon-success-vivo-new.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import {useTheme} from '../hooks';
+import {getAnimateCircleScaleInProps, getAnimateDrawLineProps} from '../utils/animation';
+
+type Props = {
+    size?: number | string;
+};
+
+const IconSuccessVivoNew = ({size = 48}: Props): JSX.Element => {
+    const {platformOverrides} = useTheme();
+
+    return (
+        <svg width={size} height={size} viewBox="0 0 64 64" fill="none">
+            <circle
+                cx="32"
+                cy="32"
+                r="24.9375"
+                stroke="url(#vivoNewGradient)"
+                strokeWidth="0.791667"
```

Comment:
weird stroke width :S but.. 🤷‍♀️ I'm going to talk with Vivo team to try to use "1px" instead of this satanic number

Reply:
Discussed offline. Updating strokeWidth to `1px` and circle radius to `25`